sdk/core/azure-core/test/ut/logging_test.cpp

```diff
@@ -3,6 +3,7 @@
 
 #include <azure/core/internal/diagnostics/log.hpp>
 
+#include <chrono>
```

Comment:
Is this needed?

Reply:
yes, we use methods from this include at the bottom of the file, somehow VS latest updaate decided it needs the include to build now , at least on my machine, preemptivelly i put it in to hopefully not cofuse the others in the team